tracer/test/snapshots/XUnitTests.SubmitTraces_packageVersion=2.6.6.verified.txt

```diff
@@ -0,0 +1,945 @@
+      git.branch: branch,
+      git.commit.author.date: AuthorDate,
+      git.commit.author.email: author@email.com,
+      git.commit.author.name: author name,
+      git.commit.committer.date: CommitterDate,
+      git.commit.committer.email: committer@email.com,
+      git.commit.committer.name: committer name,
+      git.commit.message: CommitMessage,
+      git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbb,
+      git.repository_url: repository,
+      language: dotnet,
+      library_version: LibraryVersion,
+      os.architecture: OSArchitecture,
+      os.platform: OSPlatform,
+      os.version: OSVersion,
+      runtime-id: Guid_1,
+      runtime.architecture: RuntimeArchitecture,
+      runtime.name: RuntimeName,
+      runtime.version: RuntimeVersion,
+      test.bundle: Samples.XUnitTests,
+      test.codeowners: [ "@MyTeam" ],
+      test.command: Command,
+      test.framework: xUnit,
+      test.framework_version: 2.6.6.0,
+      test.itr.forced_run: false,
+      test.itr.unskippable: false,
+      test.module: Samples.XUnitTests,
+      test.name: SimpleSkipFromAttributeTest,
+      test.skipped_by_itr: false,
+      test.skip_reason: Simple skip reason,
+      test.source.file: tracer/test/test-applications/integrations/Samples.XUnitTests/TestSuite.cs,
+      test.status: skip,
+      test.suite: Samples.XUnitTests.TestSuite,
+      test.type: test,
+      test.working_directory: CommandWorkingDirectory,
+      version: 1.0.0,
+      _dd.origin: ciapp-test
+    },
+    Metrics: {
+      process_id: 0,
+      test.source.end: 35.0,
+      test.source.start: 33.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_19,
+    SpanId: Id_20,
+    Name: xunit.test,
+    Resource: Samples.XUnitTests.TestSuite.SimpleSkipParameterizedTest,
+    Service: xunit-tests,
+    Type: test,
+    Tags: {
+      build.source_root: /project,
+      ci.workspace_path: /project,
+      env: integration_tests,
+      git.branch: branch,
+      git.commit.author.date: AuthorDate,
+      git.commit.author.email: author@email.com,
+      git.commit.author.name: author name,
+      git.commit.committer.date: CommitterDate,
+      git.commit.committer.email: committer@email.com,
+      git.commit.committer.name: committer name,
+      git.commit.message: CommitMessage,
+      git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbb,
+      git.repository_url: repository,
+      language: dotnet,
+      library_version: LibraryVersion,
+      os.architecture: OSArchitecture,
+      os.platform: OSPlatform,
+      os.version: OSVersion,
+      runtime-id: Guid_1,
+      runtime.architecture: RuntimeArchitecture,
+      runtime.name: RuntimeName,
+      runtime.version: RuntimeVersion,
+      test.bundle: Samples.XUnitTests,
+      test.codeowners: [ "@MyTeam" ],
+      test.command: Command,
+      test.framework: xUnit,
+      test.framework_version: 2.6.6.0,
```

Comment:
I don't _think_ you have scrubbing for this? 🤔 Won't this break if/when we run with a different version?

Reply:
the package version is on the name of the file.

Reply:
now I'm scrubbing this to reduce the number of snapshots.